client/controllers/incidentReports/newIncidentFromSelection.coffee

```diff
@@ -0,0 +1,79 @@
+{ buildAnnotatedIncidentSnippet } = require('/imports/ui/annotation')
+import { createIncidentReportsFromEnhancements } from '/imports/utils.coffee'
+
+POPUP_DELAY = 200
+POPUP_PADDING = 5
+POPUP_PADDING_TOP = 20
+POPUP_WINDOW_PADDING = 50
+
+_getAnnotationData = (selection) ->
+  range = selection.getRangeAt(0)
+  textOffsets = [range.startOffset, range.endOffset]
+  content = selection.anchorNode.textContent
+
+  textOffsets: textOffsets
+  text: content.slice(textOffsets[0], textOffsets[1])
+
+Template.newIncidentFromSelection.onCreated ->
+  @selection = window.getSelection()
+  @selecting = @data.selecting
+  @selecting.set(true)
```

Comment:
Sorry about these. I'll be more rigorous in looking for these sorts of references after refactoring.